frontend/src/features/workspace/ui/workspace-list.module.css

```
@@ -0,0 +1,198 @@
+.workspaceCardHeader {
+  gap: 24px;
+  padding: 34px 34px 0 !important;
+}
+
+.titleBlock {
+  display: grid;
+  gap: 18px;
+}
+
+.eyebrow {
+  font-family: "Geist Mono", "SFMono-Regular", monospace;
```

Stylelint (17.9.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)